src/Tizen.NUI.Components/Controls/TabView.cs

```diff
@@ -125,10 +179,12 @@ public void AddTab(TabButton tabButton, View view)
 
         /// <summary>
         /// Removes a tab at the specified index of TabView.
+        /// The indices of tabs(tab buttons and views) in TabView are basically the order of adding to TabView by <see cref="TabView.AddTab"/>.
+        /// So a tab(tab button and view)'s index in TabView can be changed whenever <see cref="TabView.AddTab"/> or <see cref="TabView.RemoveTab"/> is called.
```

Comment:
Please, add a missing space and consider moving "'s" to the parentheses:

```suggestion
        /// So a tab (tab button's and view's) index in TabView can be changed whenever <see cref="TabView.AddTab"/> or <see cref="TabView.RemoveTab"/> is called.
```

Reply:
Thank you for your comment.
I have updated the documentation like the other documentation as follows.
```
/// So the index of a tab(tab button and view) in TabView can be changed ...
```